docs/cost/index.md

```diff
@@ -95,6 +95,12 @@ The following mapping tables describe:
 
 #### Implementation
 
+Disclosures in the implementation section of the CoST IDS relate to changes made after the contract award to the contract value, duration and scope.
```

Comment:
> changes to a contract's value, duration or scope that were made after the contract was awarded.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/103/commits/46ff5708c157c89fd5636616b8121b77c0caec08